src/Services/Catalog/Catalog.Domain/Entities/Product.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Catalog.Domain.Entities;
+
+public class Product
+{
+    required public string Id { get; set; }
```

Comment:
why are you using string and not Guid here

Reply:
As far as I know, you need to apply [BsonId] to Id property, and I didn't want to write code connected with MongoDB in domain layer